src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxKinds.vb

```diff
@@ -23,6 +23,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
         Public ReadOnly Property EndOfLineTrivia As Integer = SyntaxKind.EndOfLineTrivia Implements ISyntaxKinds.EndOfLineTrivia
         Public ReadOnly Property SkippedTokensTrivia As Integer = SyntaxKind.SkippedTokensTrivia Implements ISyntaxKinds.SkippedTokensTrivia
         Public ReadOnly Property WhitespaceTrivia As Integer = SyntaxKind.WhitespaceTrivia Implements ISyntaxKinds.WhitespaceTrivia
+        Public ReadOnly Property SingleLineCommentTrivia As Integer = SyntaxKind.CommentTrivia Implements ISyntaxKinds.SingleLineCommentTrivia
+        Public ReadOnly Property MultiLineCommentTrivia As Integer = SyntaxKind.None Implements ISyntaxKinds.MultiLineCommentTrivia
```

Comment:
ooh, i didn't consider this bit.  i don't like the implications of this one currently.  i.e. a `default` SyntaxTrivia would count as a `MultiLineCommentTrivia` for vb... 

my knee-jerk reaction is to ask you to remove this one and have that be encoded at the feature level... but if you feel like it's totally ok, we could keep it here... i just find this very oogy (and not good precedent for other constructs both languages do not share).